### PR TITLE
RUN-2945 Worked with RVM to fix a few shape issues, added downloadRuntime API call

### PR DESCRIPTION
**Pull request by rdepena**
@StevenEBarbaro @HarsimranSingh @datamadic @whyn07m3 check it out.

[Test resuts](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/593f017e34e44123d883e941)

[Vanilla 5.49.21.12 results](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/593f044134e44123d883e942)

### CODE REVIEW

File: src/browser/rvm/rvm_message_bus.ts [hunk 1]
```diff
@@ -161,7 +176,7 @@ export interface LicenseInfo {
 /**
  * Module to facilitate communication with the RVM.
  * A transport can be passed in to be used, otherwise a new WMCopyData transport is used.
- * 'broadcast' messages received from RVM(RVM initiated) will be broadcasted
+  * 'broadcast' messages received from RVM(RVM initiated) will be broadcasted
```
Comment (HarsimranSingh) [inline src/browser/rvm/rvm_message_bus.ts]: :neckbeard: :neckbeard: remove unnecessary space

File: src/browser/api_protocol/api_handlers/system.js [hunk 2]
```diff
@@ -335,18 +336,28 @@ function SystemApiHandler() {
...
                 ack(dataAck);
             } else {
                 errorAck(err);
             }
         });
     }
 
+    function downloadRuntime(identity, message, ack, nack) {
+        const { payload } = message;
+        const dataAck = Object.assign({}, successAck);
```
Comment (HarsimranSingh) [inline src/browser/api_protocol/api_handlers/system.js]: Should we keep this as `_.clone()` for consistency?

Comment (rdepena) [inline src/browser/api_protocol/api_handlers/system.js]: that's a good point, I can use clone here and setup a task to move those to assign.

File: src/browser/rvm/rvm_message_bus.ts (hunk 1 above)
Comment (rdepena) [inline src/browser/rvm/rvm_message_bus.ts]: good catch.

File: src/browser/api_protocol/api_handlers/system.js (hunk 2 above)
Comment (HarsimranSingh) [inline src/browser/api_protocol/api_handlers/system.js]: 👍

**Pull request merged**